### PR TITLE
Address service team's feedback on samples

### PR DESCRIPTION
**Pull request by mssfang**
- fixed typo based on the service team's feedback.  
- use %.2f instead of %s for double values.
- phrases are all printed in a separate line with the "recognized phrase" as a prefix, but should not
- We are not printing matches in LinkedEntities output.

fixes: #8122

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/AnalyzeSentimentBatchDocuments.java [hunk 1]
```diff
@@ -30,11 +30,11 @@
     public static void main(String[] args) {
         // Instantiate a client that will be used to call the service.
         TextAnalyticsClient client = new TextAnalyticsClientBuilder()
-            .apiKey(new TextAnalyticsApiKeyCredential("{api_key}"))
-            .endpoint("{endpoint}")
+            .apiKey(new TextAnalyticsApiKeyCredential("b2f8b7b697c348dcb0e30055d49f3d0f"))
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/AnalyzeSentimentBatchDocuments.java]: Revert the keys and endpoint information from here

File: sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/RecognizePiiAsync.java [hunk 2]
```diff
@@ -24,12 +24,12 @@ public static void main(String[] args) {
             .endpoint("{endpoint}")
             .buildAsyncClient();
 
-        // The text that need be analysed.
+        // The text that need be analyzed.
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/RecognizePiiAsync.java]: The text that needs to be analyzed.

File: sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/AnalyzeSentimentBatchDocuments.java (hunk 1 above)
Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/AnalyzeSentimentBatchDocuments.java]: Yup. Found this when reviewed. Thanks for pointing this out

Comment (maririos) [reply sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/AnalyzeSentimentBatchDocuments.java]: we need to rotate that key too

File: sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java [hunk 3]
```diff
@@ -108,8 +109,19 @@ public void recognizePiiEntity() {
...
+            System.out.println("Linked Entities:");
+            System.out.printf("Name: %s, ID: %s, URL: %s, data source: %s.%n",
+                linkedEntity.getName(),
+                linkedEntity.getId(),
+                linkedEntity.getUrl(),
+                linkedEntity.getDataSource());
+            for (LinkedEntityMatch linkedEntityMatch : linkedEntity.getLinkedEntityMatches()) {
+                System.out.printf("Text: %s, offset: %s, length: %s, score: %.2f.%n",
+                    linkedEntityMatch.getText(),
+                    linkedEntityMatch.getOffset(),
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java]: We can consider removing these from the example. It makes the example look very lengthy.

File: sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/RotateApiKeyAsync.java [hunk 4]
```diff
@@ -35,8 +35,9 @@ public static void main(String[] args) {
         // Update the API key
         credential.updateCredential("{valid_api_key}");
 
+        System.out.println("Recognized phrases:");
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/RotateApiKeyAsync.java]: nit: Extracted phrases, may be to match what with the rest.

**Pull request merged**